packages/shared/prisma/schema.prisma

```diff
@@ -1433,3 +1434,20 @@ enum SurveyName {
 
   @@map("SurveyName")
 }
+
+model CloudSpendAlert {
```

Comment:
I think accepting a list of strings here to customize the recipients would be useful. Can be an extension if customers ask for it.

Reply:
would move this out-of-scope for now. If someone asks we can ship it